SFS_monochromatic/driving_functions_mono/driving_function_mono_localwfs_sbl_ps.m

```diff
@@ -107,20 +107,24 @@
 Dwfs = zeros(N0,1);
 Dwfs(xdx) = driving_function_mono_wfs(x0(xdx,:),xs,'ps',f,conf);
 % Coefficients for lowpass filtering of WFS Driving signal
-[zlp,plp,klp] = linkwitz_riley(Nlr,wc,'low','s');  % LR Lowpass filter
+% LR Lowpass filter (normalised cutoff-frequency)
+[zlp,plp,klp] = linkwitz_riley(Nlr,1,'low','s');
 % Lowpass filtering
 [sos,g] = zp2sos(zlp,plp,klp,'down','none');  % generate sos
-Hlp = g.*prod( (sos(:,1:3)*[-omega.^2; 1i*omega; 1]) ...
-      ./ (sos(:,4:6)*[-omega.^2; 1i*omega; 1]),1);
+if isoctave
+  % WORKAROUND: there seems to be an Octave bug
+  Hlp = g.*prod( (sos(:,3:-1:1)*omega_vec)./(sos(:,4:6)*omega_vec),1);
+else
+  Hlp = g.*prod( (sos(:,1:3)*omega_vec)./(sos(:,4:6)*omega_vec),1);
+end
```

Comment:
Is this to deal with https://savannah.gnu.org/bugs/?51936?

Reply:
Yes, dunno, if there exists a better workaround.

Reply:
I don't have a better one. I will add the link to the bug in the comment there.
Is this otherwise ready for clean up, or are you planing to push more to this PR?

Reply:
ready for clean up